.github/workflows/swift-toolchain.yml

```diff
@@ -342,6 +344,8 @@ jobs:
     needs: [cmark_gfm]
     runs-on: ${{ inputs.default_build_runner }}
 
+    name: Windows amd64 Compiler Build Tools
```

Comment:
Could this be called "build-arch" instead of "amd64" even though it's hardcoded at this point?

Reply:
I'd love to make that a matrix field, but right now, we hardcode build to amd64 as GH does not provide non-x64 build hosts.